src/core/db/repository/base.py

```diff
@@ -104,7 +105,10 @@ async def archive_by_ids(self, ids: list[int]) -> None:
             .values({"is_archived": True})
         )
 
-    async def get_by_ids(self, ids: list[int]) -> list[int]:
+    async def get_by_ids(self, ids: list[int]) -> Sequence[int]:
         """Возвращает id объектов модели из базы данных по указанным ids"""
-        filtered_ids = await self._session.scalars(select(self._model.id).where(self._model.id.in_(ids)))
-        return filtered_ids.all()
+        filtered_ids = await self._session.scalar(

```

Comment:
```suggestion
        filtered_ids = await self._session.scalars(
```

Почему `scalar`? Будь по внимательнее) Не забудь протестировать.